packages/celotool/src/cmds/account/verify.ts

```diff
@@ -51,7 +54,12 @@ async function verifyCmd(argv: VerifyArgv) {
   const attestations = await kit.contracts.getAttestations()
   await printCurrentCompletedAttestations(attestations, argv.phone, account)
 
-  let attestationsToComplete = await attestations.getActionableAttestations(argv.phone, account)
+  let currentBlockNumber = await kit.web3.eth.getBlockNumber()
+  let attestationsToComplete = await attestations.getActionableAttestations(
+    argv.phone,
+    account,
+    currentBlockNumber
```

Comment:
Why does this take the block number? Can it instead fetch the block number itself?

Reply:
I thought it would be nice for consumers to specify a target block for when the expiry is about the corner, but maybe that is premature